spec/models/services/shopping_cart_services/recalculate_item_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# rubocop:disable RSpec/MultipleExpectations
+
+RSpec.describe ShoppingCartServices::RecalculateItem, type: :model do
```

Comment:
cala ta specka to karna koza! :D IMO nie miales tutaj powodu wylaczac ten check. Plus jesli juz wylaczasz copa to daj komentarz dlaczego. Oszczedzisz tym sposobem wszystkim czasu :)

Reply:
Zrobiłem refactor specek i serwisów. Nie wiem czy nie czeka mnie grubszy refactor i czy gdzieś nie poszedłem w złą stronę...